app/views/lightweight_activities/_edit_header.html.haml

```diff
@@ -4,9 +4,18 @@
       = svg_icon(asset_path "/assets/images/icons/page-nav/previous.svg")
     %button{ :id => "page-nav-home", :class => "page-button current", :onclick => "window.location='#{edit_activity_path(@activity)}'" }
       = svg_icon(asset_path "/assets/images/icons/page-nav/home.svg")
+    - hidden_pages_before = 0
     - @activity.pages.each.with_index(1) do |p, index|
-      %button{ :id => dom_id_for(p, :item), :class => "page-button #{p.is_completion ? 'completion-page-button' : ''}", :onclick => "window.location='#{edit_activity_page_path(@activity, p)}'" }
-        = p.is_completion ? svg_icon(asset_path "/assets/images/icons/page-nav/completion.svg") : index
+      - special_class = p.is_hidden ? 'hidden-page-button' : ''
+      - special_class = p.is_completion ? 'completion-page-button' : ''
+      %button{ :id => dom_id_for(p, :item), :class => "page-button #{special_class}", :onclick => "window.location='#{edit_activity_page_path(@activity, p)}'" }
+        - if p.is_hidden

```

Comment:
Not sure if we need to do this.  I think we could use this method too:
on the page model there is a `page_number` method...
https://github.com/concord-consortium/lara/blob/master/app/models/interactive_page.rb#L232

Reply:
Oh, thanks. I'll use `page_number`. Seems like the best thing to do.